src/repositories/page_repository.rs

```diff
@@ -71,6 +71,17 @@ impl PageRepository {
         model_count
     }
 
+    pub async fn remove_by_id(
+        &self,
+        datastore: &Datastore,
+        database_session: &Session,
+        page_id: &String) -> Result<bool>
+    {
+        let sql = format!("DELETE pages:{page_id}");
+        datastore.execute(&sql, database_session, None).await?;
+        Ok(true)
+    }
+
```

Comment:
In this file change the Line no :104 

    let result_object = match result_object_option {
            Some(object) => object,
            None => Err(Error::NotFound("no record found".to_string())),
        };